src/shared/components/msktable/MSKTable.tsx

```diff
@@ -18,12 +21,20 @@ type MSKTableProps<T> = {
 
 class MSKTableStore<T> {
     @observable public filterString:string;
-    @observable public page:number;
+    @observable private _page:number;
     @observable public itemsPerPage:number;
     @observable public sortColumn:string;
     @observable public sortAscending:boolean;
     @observable public columns:Column<T>[];
```

Comment:
I like it better as an array here, since that also encodes their order. I think the manipulation of columns should happen at a higher layer.